### PR TITLE
Feature/map prototype

### LINKED ISSUE #4: Preparation for map view
### THIS IS A RESEARCH TASK

Users want to see their route on a map 
to get a better impression of their vacation.
In order to do so, a map API needs to be implemented. Gather information on how to do this, so a new feature can be added next week.
----
Acceptance criteria:
- [x] Make yourself familiar with a map API
- [x] Try to set up a prototype (show pin on map based on entry on a form)

### PR DESCRIPTION
**Pull request by derjubu**
Fixes #4 
Implement map on a page. The startlocation on the map cannot be changed yet, but it is scrollable/zoomable.
This is a technical ticket as preparation for features next week.
https://capstone-rev-feature-ma-lqniyr.herokuapp.com/map

### CODE REVIEW

File: src/app/components/Map/Map.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  if (typeof import.meta.env.VITE_MAPBOX_ACCESS_KEY === 'string') {
+    mapboxgl.accessToken = import.meta.env.VITE_MAPBOX_ACCESS_KEY;
+  } else {
+    throw new Error('no KEY provided');
+  }
+
+  const mapContainer = useRef<HTMLDivElement | null>(null);
+  const map = useRef<null | Map>(null);
+  const [lng, setLng] = useState<number>(10.0125);
+  const [lat, setLat] = useState<number>(53.5469);
```
Comment (lajanzen) [inline src/app/components/Map/Map.tsx]: I'd recommend not working with abbreviations here.

File: src/app/components/Map/Map.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
+import React, { useRef, useEffect, useState } from 'react';
+import mapboxgl from 'mapbox-gl';
+import type { Map } from 'mapbox-gl';
+import 'mapbox-gl/dist/mapbox-gl.css';
+import styled from 'styled-components';
+
+export default function LocationMap(): JSX.Element {
```
Comment (julialoeschel) [inline src/app/components/Map/Map.tsx]: I think the name of the file shoul match the function name (at least thats what i remember)

File: src/app/components/Map/Map.tsx [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+      <MapLegend>
+        Longitude: {lng} | Lattitude: {lat} | Zoom: {zoom}
+      </MapLegend>
+      <MapContainer ref={mapContainer} />
+    </div>
+  );
+}
+const MapContainer = styled.div`
+  height: 400px;
+  margin: 2px;
```
Comment (lajanzen) [inline src/app/components/Map/Map.tsx]: I'd suggest working with relative sizes instead with px.

**Review by julialoeschel — COMMENTED** [review]
great work!!

**Pull request merged**